Tutorials/intro-to-octave/configuration.py

```diff
@@ -5,27 +5,10 @@
 ######################
 qop_ip = "127.0.0.1"
 qop_port = 80
-
```

Comment:
This needs to be replaced by 
```
qop_ip = "127.0.0.1"  # Write the QM router IP address
cluster_name = None  # Write your cluster_name if version >= QOP220
qop_port = None  # Write the QOP port if version < QOP220

```

Reply:
Are we changing all of the script to be with the cluster name?

Reply:
Yes, but they should already have cluster_name everywhere in the SC folder